WordPress/src/androidTest/java/org/wordpress/android/e2e/ReaderTests.java

```diff
@@ -0,0 +1,63 @@
+package org.wordpress.android.e2e;
+
+import android.Manifest.permission;
+
+import androidx.test.rule.ActivityTestRule;
+import androidx.test.rule.GrantPermissionRule;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.wordpress.android.e2e.pages.ReaderPage;
+import org.wordpress.android.support.BaseTest;
+import org.wordpress.android.ui.WPLaunchActivity;
+
+
+public class ReaderTests extends BaseTest {
+    @Rule
+    public ActivityTestRule<WPLaunchActivity> mActivityTestRule = new ActivityTestRule<>(WPLaunchActivity.class);
+
+    @Rule
+    public GrantPermissionRule mRuntimeImageAccessRule = GrantPermissionRule.grant(permission.WRITE_EXTERNAL_STORAGE);
+
+    @Before
+    public void setUp() {
+        logoutIfNecessary();
```

Comment:
Super nitpick -  Probably existing code, but seems like there are two words to describe the same condition: `logoutIfNecessary` and from line 33 below `dismissReaderViewIfNeeded`. We should stick with one for a bit of consistency.

Reply:
Good catch! I'll stick to the `Necessary` just because it was already there. Thanks! =)